docs/source/index.rst

```diff
@@ -24,15 +23,37 @@ NuCypher
     :alt: Discord
 
 .. image:: https://readthedocs.org/projects/nucypher/badge/?version=latest
-    :target: https://nucypher.readthedocs.io/en/latest/?badge=latest
+    :target: https://nucypher.readthedocs.io/en/latest/
     :alt: Documentation Status
 
 .. image:: https://img.shields.io/pypi/l/nucypher.svg
     :target: https://www.gnu.org/licenses/gpl-3.0.html
 
+.. _Umbral: https://github.com/nucypher/pyUmbral
+
+The NuCypher network uses the Umbral_ threshold proxy re-encryption scheme
+to provide cryptographic access controls for distributed apps and protocols.
+
+1. Alice, the data owner, grants access to her encrypted data to
+anyone she wants by creating a policy and uploading it to
+the NuCypher network.
+
+2. Anyone can encrypt data using Alice's policy public key.
+The resulting encrypted data can be uploaded to IPFS, Swarm, S3,
```

Comment:
``` suggestion
The resulting encrypted data can be uploaded to a storage layer like IPFS, Swarm, S3,
```

Reply:
The next line said: ``or any other storage layer.``
Is that OK?